src/core/DY.Core.Bytes.Type.fst

```diff
@@ -18,27 +18,27 @@ open DY.Core.Label.Type
 /// To this bytestring will correspond a usage (e.g. "SigKey"),
 /// which will ensure it can only be used as the key of one specific cryptographic primitive (e.g. signature function).
 ///
-/// Second, the various usages additionally contain a usage string,
+/// Second, the various usages additionally contain a usage string "tag",
 /// which is used to further distinguish keys used with a given cryptographic primitive:
 /// for example to distinguish the long-term key from the ephemeral ones,
 /// or to distinguish keys from two sub-protocols running in parallel.
 /// At the end, this usage string may play a role in the security proof
 /// via the protocol invariants.
 ///
-/// Although most usages only contain a usage string to disambiguate different keys,
-/// KDF keys additionally contains a full `bytes` in their usage.
+/// Because a string may not be enough to express the complex reasons two keys are distinct,
+/// usages also contain a `bytes` called "data".
 /// This is useful to do security proofs of protocols with complex key schedules,
 /// allowing the usage's `data` field and the label of the keys to evolve
 /// when calling kdf_extract and kdf_expand.
 
 type usage =
   | NoUsage: usage // baked-in None
-  | SigKey: tag:string -> usage
+  | SigKey: tag:string -> data:bytes -> usage
   | SigNonce: usage
-  | PkdecKey: tag:string -> usage
+  | PkdecKey: tag:string -> data:bytes -> usage
```

Comment:
Nit: `PkDecKey` seems like a more correct capitalization here (and if we're using `SigKey` for signatures, `PkKey` or something similar might match better)